src/app/app.routes.ts

```diff
@@ -0,0 +1,3 @@
+import { Routes } from '@angular/router';
```

Comment:
suggestion

          import { Routes } from '@angular/router'; // Importing Routes from Angular Router package

          Explantion: Adding a comment for clarity on what is being imported.